python/ql/lib/semmle/python/dataflow/new/TestSummaries.qll

```diff
@@ -0,0 +1,123 @@
+private import python
```

Comment:
![image](https://i.imgur.com/iw4YuOi.gif)

I've seen this file before... is this not a copy of an existing file? Why do we need it twice? it does not seem to be used.

Reply:
ahh, I see this file was deleted again in a later commit.